.github/workflows/main.yml

```diff
@@ -20,6 +20,8 @@ jobs:
           python -m pip install tox
       - name: Run tests
         run: tox
+      - name: Upload coverage to Codecov
+        run: tox -e coverage,codecov
```

Comment:
Could you use https://github.com/codecov/codecov-action instead?

Reply:
Sure, thanks.